tests/Admin/ElectedRepresentative/ElectedRepresentativeAdminTest.php

```diff
@@ -0,0 +1,111 @@
+<?php
+
+namespace Tests\App\Controller\Admin\ElectedRepresentative;
+
+use App\DataFixtures\ORM\LoadElectedRepresentativeData;
+use App\Entity\ElectedRepresentative\ElectedRepresentative;
+use App\Mailchimp\Synchronisation\Command\ElectedRepresentativeArchiveCommand;
+use App\Mailchimp\Synchronisation\Command\ElectedRepresentativeChangeCommand;
+use App\Repository\ElectedRepresentative\ElectedRepresentativeRepository;
+use Liip\FunctionalTestBundle\Test\WebTestCase;
+use Tests\App\Controller\ControllerTestTrait;
+use Tests\App\MessengerTestTrait;
+
+/**
+ * @group functional
+ * @group admin
+ */
+class ElectedRepresentativeAdminTest extends WebTestCase
+{
+    private const EDIT_URI_PATTERN = '/admin/app/electedrepresentative-electedrepresentative/%s/edit';
```

Comment:
is it normal that `electedrepresentative` is doubled in the url ?

Reply:
yes. It's because Sonata constructs its url based on the path of the entity.

Here it is `src/ElectedRepresentative/ElectedRepresentative.php`

and here you have your double string